packages/react-vapor/src/components/table-hoc/TableWithFilter.tsx

```diff
@@ -63,14 +60,10 @@ export const tableWithFilter = (
             filter: filterText,
             urlFilter: urlParams[Params.filter],
             data: ownProps.isServer || config.isServer ? ownProps.data : ownProps.data && filterData(),
-        };
+        } as any;
     };
 
-    const mapDispatchToProps = (dispatch: IDispatch, ownProps: ITableHOCOwnProps) => ({
-        resetFilter: () => dispatch(filterThrough(ownProps.id, '')),
-    });
-
-    @ReduxConnect(mapStateToProps, mapDispatchToProps)
+    @ReduxConnect(mapStateToProps)

```

Comment:
`const TableWithFilter = connect(mapStateToProps)(TableWithFilterDisconnected)`  since we dont use the `@ReduxConnect` anymore ?